README.md

```diff
@@ -276,6 +281,82 @@ TurkishCities.find_by_postcode(34382)    # => "Couldn't find any subdistrict wit
 TurkishCities.find_by_postcode('100000') # => Given value [100000] is outside bounds of 1010 to 81952.
 ```
 
+### Finding city name with population data
+
+### Exact search
+
+Search city name with exact population
+
+```rb
+TurkishCities.find_by_population('exact', 15840900)
+# => ["İstanbul"]
+TurkishCities.find_by_population('exact', 898369)
+# => ["Eskişehir"]
+TurkishCities.find_by_population('exact', 2130432)
+# => ["Gaziantep"]
+TurkishCities.find_by_population('exatc', 2130432)
+# => "Population type 'exatc' is unsupported"
+TurkishCities.find_by_population('exact', 10432)
```

Comment:
Simply returning an empty array would be better than throwing an error when no matches found.